ncm-metaconfig/src/main/metaconfig/singularity/pan/schema.pan

```diff
@@ -0,0 +1,118 @@
+    'setuid' : boolean = true
+    'pid' : type_singularity_allow_ns
+} = dict();
+
+type type_singularity_user = {
+    @{If /etc/passwd exists within the container, this will automatically append
+    an entry for the calling user}
+    'passwd' : boolean = true
+    @{If /etc/group exists within the container, this will automatically append
+    an entry for the calling user}
+    'group' : boolean = true
+    @{If there is a bind point within the container, use the host's /etc/resolv.conf}
+    'resolv_conf' : boolean = true
+} = dict();
+
+type type_singularity_overlay = {
+    @{Enabling this option will make it possible to specify bind paths to locations
+    that do not currently exist within the container. Some limitations still exist
+    when running in completely non-privileged mode. (note: this option is only
+    supported on hosts that support overlay file systems)}
+    'overlay' : boolean = false
+} = dict();
+
+type type_singularity_mount = {
+    @{Should we automatically bind mount /proc within the container?}
+    'proc' : boolean = true
+    @{Should we automatically bind mount /sys within the container?}
+    'sys' : boolean = true
+    @{Should we automatically bind mount /dev within the container? If you select
+    minimal, and if overlay is enabled, then Singularity will attempt to create
+    the following devices inside the container: null, zero, random and urandom}
+    'dev' : boolean = true
+    @{Should we automatically determine the calling user's home directory and
+    attempt to mount it's base path into the container? If the --contain option
+    is used, the home directory will be created within the session directory or
+    can be overridden with the SINGULARITY_HOME or SINGULARITY_WORKDIR
+    environment variables (or their corresponding command line options)}
+    'home' : boolean = true
+    @{Should we automatically bind mount /tmp and /var/tmp into the container? If
+    the --contain option is used, both tmp locations will be created in the
+    session directory or can be specified via the  SINGULARITY_WORKDIR
+    environment variable (or the --workingdir command line option)}
+    'tmp' : boolean = true
+    @{Probe for all mounted file systems that are mounted on the host, and bind
+    those into the container?}
+    'hostfs' : boolean = false
+    @{Should we automatically propagate file-system changes from the host?
+    This should be set to 'true' when autofs mounts in the system should
+    show up in the container}
+    'slave' : boolean = true
+} = dict();
+
+type type_singularity_bind_path = {
+    @{Define a list of files/directories that should be made available from within
+    the container. The file or directory must exist within the container on
+    which to attach to. you can specify a different source and destination
+    path (respectively) with a colon; otherwise source and dest are the same}
+    'path' ? string[]
```

Comment:
you need to make a type here that allows `/abc:/def`; so absolute filename forllowed by optional absolute filename